tests/php/test-class-wc-connect-order-presenter.php

```diff
@@ -8,6 +8,25 @@ public static function setupBeforeClass() {
 		WC_Connect_Compatibility::set_version( '3.0.0' );
 	}
 
+	public function test_get_order_for_api_null_values() {
+		// Setup order
+		$order = WC_Helper_Order::create_order();
+		$order->save();
```

Comment:
`WC_Helper_Order::create_order()` is already calling `save()` in the order, but it seems saving is not necessary here as we're just using a `WC_Order` object.